interface/interface.dm

```diff
@@ -256,16 +256,11 @@ Any-Mode: (hotkey doesn't need to be on)
 	set desc = "View the current map in the webviewer"
 	set hidden = 1
 	var/map_in_url
-	switch(SSmapping.config?.map_name)
-		if("Box Station")			map_in_url = "box"
-		if("Delta Station")			map_in_url = "delta"
-		if("Donutstation")			map_in_url = "donut"
-		if("MetaStation")			map_in_url = "meta"
-		if("Kilo Station")          map_in_url = "kilo"
-		if("PubbyStation")          map_in_url = "pubby"
+	switch(SSmapping.config?.map_name)																	// NSV Changes begin
+		if("NSV Hammerhead")			map_in_url = "Hammerhead"
 	if(map_in_url)
-		if(alert("This will open the current map in your browser. Are you sure?",,"Yes","No")!="Yes")
+		if(alert("This will open the current map in your browser. Are you sure?",,"Yes","No")!="Yes")	// NSV Changes end

```

Comment:
This tag needs to go to Line 264, and this entire bit of code is awful and should load from the map json instead of being hardcoded

Reply:
I have basically no clue on how I'd start that.